### PR TITLE
Logging updates

### PR DESCRIPTION
**Pull request by sprak3000**
* PSR-2 formatting and other small code polish
* Make the logging statements less generic by using the same message from the exceptions thrown.

### CODE REVIEW

File: lib/TraackrApi/TraackrApiObject.php [hunk 1]
```diff
@@ -2,256 +2,299 @@
...
 
-      if ( is_bool($bool) ) return $bool ? 'true' : 'false';
+        if ($httpcode != '200') {
+            $info = curl_getinfo($this->curl);
 
-      if ( strtolower($bool) === 'true' ) return 'true';
+            if ($httpcode == '400') {
+                // Let's try to see if it's a bad customer key
+                if ($curl_exec === 'Customer key not found') {
+                    $message = 'Invalid Customer Key (HTTP 400)';
```
Comment (sprak3000) [inline lib/TraackrApi/TraackrApiObject.php]: Not sure why the diff is such a hot mess, but this is the pattern of the logging used. Capture the overall message and use that message in the logger call and exception thrown.

File: lib/TraackrApi/TraackrApiObject.php [hunk 2]
```diff
@@ -2,256 +2,299 @@
```
Comment (sprak3000) [inline lib/TraackrApi/TraackrApiObject.php]: Set the diff settings to hide whitespace changes. Will be much easier to go through that way.

Comment (KidA78) [reply lib/TraackrApi/TraackrApiObject.php]: thanks for the tip, made the difference

File: lib/TraackrApi/TraackrApiObject.php (hunk 1 above)
Comment (KidA78) [reply lib/TraackrApi/TraackrApiObject.php]: So functionally nothing changes, but we now have error messages that are consistent between the exception thrown and what the logger records.... am I understanding this correctly?

Comment (sprak3000) [reply lib/TraackrApi/TraackrApiObject.php]: You are understanding this correctly.

**Pull request merged**